pkg/cliplugins/workload/plugin/sync.go

```diff
@@ -147,18 +148,18 @@ func getSyncerID(syncTarget *workloadv1alpha1.SyncTarget) string {
 // enableSyncerForWorkspace creates a sync target with the given name and creates a service
 // account for the syncer in the given namespace. The expectation is that the provided config is
 // for a logical cluster (workspace). Returns the token the syncer will use to connect to kcp.
-func (c *Config) enableSyncerForWorkspace(ctx context.Context, config *rest.Config, syncTargetName, namespace string) (string, string, error) {
+func (c *Config) enableSyncerForWorkspace(ctx context.Context, config *rest.Config, syncTargetName, namespace string) (string, string, string, error) {
```

Comment:
named return values woul dmake this much easier to read.

Reply:
the implicit behavior is very yuck though ;)